src/containers/walletContainer.js

```diff
@@ -149,6 +149,10 @@ class WalletContainer extends Component {
                 open: false
             })
         }
+        this.setState({
```

Comment:
Please check my comment on this in your other PR. Merging this PR will result in a merge conflict as this code already exists in your other PR. Please resolve this.